main/commons/src/main/java/org/cryptomator/common/settings/VaultSettings.java

```diff
@@ -38,7 +31,7 @@
 	public static final int DEFAULT_FILENAME_LENGTH_LIMIT = -1;
 	public static final WhenUnlocked DEFAULT_ACTION_AFTER_UNLOCK = WhenUnlocked.ASK;
 	public static final boolean DEFAULT_LOCK_AFTER_TIME = false;
-	public static final String DEFAULT_LOCK_TIME_IN_MINUTES = "30";
+	public static final int DEFAULT_LOCK_TIME_IN_MINUTES = 30;
```

Comment:
I have a slight preference towards SI units. In other words: Use seconds internally. If that's not too much hassle with the conversion for UI.